### PR TITLE
Added unit tests for inventory display.

### PR DESCRIPTION
**Pull request by ren-tham**

### CODE REVIEW

File: projects/storefrontlib/src/cms-components/cart/add-to-cart/add-to-cart.component.spec.ts [hunk 1]
```diff
@@ -10,18 +10,34 @@ import {
...
+  CmsAddToCartComponent
 } from '@spartacus/core';
 import { BehaviorSubject, Observable, of } from 'rxjs';
 import { ModalService } from '../../../shared/components/modal/index';
 import { SpinnerModule } from '../../../shared/components/spinner/spinner.module';
 import { CurrentProductService } from '../../product';
+import { CmsComponentData } from '../../../cms-structure/page/model/cms-component-data';
 import { AddToCartComponent } from './add-to-cart.component';
 
+
```
Comment (mszczepaniak) [inline projects/storefrontlib/src/cms-components/cart/add-to-cart/add-to-cart.component.spec.ts]:
```suggestion
```

File: projects/storefrontlib/src/cms-components/cart/add-to-cart/add-to-cart.component.spec.ts [hunk 2]
```diff
@@ -250,4 +272,108 @@ describe('AddToCartComponent', () => {
     const quantityEl = el.query(By.css('.quantity'));
     expect(quantityEl).toBeNull();
   });
+
+
```
Comment (mszczepaniak) [inline projects/storefrontlib/src/cms-components/cart/add-to-cart/add-to-cart.component.spec.ts]:
```suggestion
```

File: projects/storefrontlib/src/cms-components/cart/add-to-cart/add-to-cart.component.spec.ts [hunk 3]
```diff
@@ -250,4 +272,108 @@ describe('AddToCartComponent', () => {
...
+
+  describe('Inventory Display test', () => {
+
+    it('should display inventory quantity when enabled', () => {
+
+      const currentMockProduct: Product = {
```
Comment (mszczepaniak) [inline projects/storefrontlib/src/cms-components/cart/add-to-cart/add-to-cart.component.spec.ts]: let's put currentMockProduct in the describe above and reuse it for every it case

File: projects/storefrontlib/src/cms-components/cart/add-to-cart/add-to-cart.component.spec.ts [hunk 4]
```diff
@@ -72,7 +86,7 @@ class MockItemCounterComponent {
   @Input() control;
 }
 
-describe('AddToCartComponent', () => {
+fdescribe('AddToCartComponent', () => {
```
Comment (bgambocjaviniar) [inline projects/storefrontlib/src/cms-components/cart/add-to-cart/add-to-cart.component.spec.ts:89]: remove fdescribe

**Review by bgambocjaviniar — CHANGES_REQUESTED** [review]
https://github.com/SAP/spartacus/pull/11978#pullrequestreview-696581686
see point # 3
i'd prefer this file gets included in that PR above, and to adapt to the fixes asked in the PR if necessary

**Pull request merged**